### DISCUSSION TITLE
[远期] 加添冯版的分类

### DISCUSSION THREAD

**Discussion by Guanchishan** [MindongLab/yngdieng#216] [产品功能 ideas]
提议 by @DDFShanyu 
![image](https://user-images.githubusercontent.com/30859354/79366707-3e3d4b80-7f7f-11ea-89d6-e659e8225ad9.png)
![image](https://user-images.githubusercontent.com/30859354/79366715-41d0d280-7f7f-11ea-9c8b-7dcffeb8ff5b.png)

> 子善  0:51:36
关于词汇分类，可以按冯书后面的分类标准来
但是分类工程量非常大

关于优先程度，子善意见：

> 我觉得分类不急
[关键词](#88)我觉得比这个要紧一些
以前厝边有分类，但是我觉得没什么用，会去看的人很少，而且分类里面的逻辑也比较混乱

批注：我以前会看hhh

> 比如有乇配，是放在食物呢，还是日常生活呢，还是动词呢
contrib我觉得可以先分着试试看，但是其他书的分类我们先不动，因为人手不足

子善指出，

> 词汇分类，最系统的就是按语法来分，但是就算是这种分类，从古典语文学吵到现在的生成语法，都没有产生一个定论

而冯版分类

> 冯书那种方言调查式的分类我觉得不错
方言学界为了方便田野调查有比较通用的分法
就是经常可以看到的 词表一进去就是风雨雷电的那种

因为以前的纸质书不可能做小爱同学或天猫精灵，你对他喊一声“把快乐翻译成福州话”他不会理你，所以才需要方言调查式分类以资查考义类。现在借助数位科技可能确实加“关键词”会更直接。但是分类毕竟有其经纬作用。这种方言调查式的分类若引入相信能够方便搜索。

**Comment by Guanchishan** [comment]
分类可以采用词性与方言调查式分类并行的方式？

**Comment by Guanchishan** [comment]
冯版分类手录如下：

天文

- 日月星
- 風雲雷雨
- 冰雪霜露
- 氣候

地理

- 地
- 山
- 江河湖海水
- 石沙土塊礦物
- 城鄉處所

時令時間

- 季節
- 節日
- 年
- 月
- 日時
- 其他時間概念

農業

- 農事
- 農具

植物

- 農作物
- 豆類菜蔬
- 樹木
- 瓜果
- 花草菌類

動物

- 牲畜
- 鳥獸
- 蟲類
- 魚蝦類

房舍

- 房子
- 房屋結構
- 其他設施

器具用品

- 一般傢俱
- 臥室用具
- 炊事用具
- 工匠用具
- 其他生活用品

稱謂

- 一般稱謂
- 職業稱謂

親屬

- 長輩
- 平輩
- 晚輩
- 其他

身體

- 五官
- 手腳胸背等
- 其他

疾病醫療

- 一般用語
- 內科
- 外科
- 殘疾等

衣服穿戴

- 服裝
- 鞋帽
- 裝飾品
- 其他穿戴用品

飲食

- 伙食
- 米食
- 麵食
- 肉蛋
- 菜
- 油鹽作料
- 煙茶酒

紅白大事

- 婚姻生育
- 壽辰喪葬
- 宗教等

日常生活

- 衣
- 食
- 住
- 行
- 其他

訟事
交際
商業交通

- 經商行業
- 經營交易
- 賬目度量衡
- 交通

文化教育

- 學校
- 教室文具
- 讀書識字
- 寫字

文體活動

- 遊戲玩具
- 體育
- 武術舞蹈
- 戲劇

動作

- 一般動作
- 心理活動
- 語言動作

位置
代詞等
形容詞
副詞介詞等
量詞
附加成分
數字等
其他

但是榕典引入这套分类树时，可能要注意：

1. 一些分类，如交通商业，是可以拆开成为交通与商业的；
1. “其他”性质的分类原则上不能存在，这才是导致分类紊乱的关键。可能可以把其他的词汇归并到上级分类，如“其他生活用品”直接并到上级分类“器具用品”下
1. 词性、词义、情感义分类的混乱有待解决。

**Comment by Guanchishan** [comment]
尝试整理如下（未来继续删改）：

**詞義**

天文
- 日
- 月
- 星
- 風
- 雲
- 雷
- 雨
- 冰雪霜露
- 氣候

地理

- 地
- 山
- 江河湖海水
- 石沙土塊礦物
- 城鄉處所

時令時間

- 季節
- 節日
- 年
- 月
- 日時
- 其他時間概念

農業

- 農事
- 農具

植物

- 農作物
- 豆類菜蔬
- 樹木
- 瓜果
- 花草
- 菌類

動物

- 牲畜
- 鳥獸
- 蟲類
- 魚蝦類

房舍

- 房子
- 房屋結構
- 其他設施

器具用品

- 一般傢俱
- 臥室用具
- 炊事用具
- 工匠用具
- 其他生活用品

稱謂

- 一般稱謂
- 職業稱謂

親屬

- 長輩
- 平輩
- 晚輩
- 其他親屬

身體

- 五官
- 手腳胸背等
- 其他身體部位

疾病醫療

- 一般醫療用語
- 內科
- 外科
- 殘疾等

衣服穿戴

- 服裝
- 鞋帽
- 裝飾品
- 其他穿戴用品

飲食

- 伙食
- 米食
- 麵食
- 肉蛋
- 菜
- 油鹽作料
- 煙茶酒

紅白大事

- 婚姻生育
- 壽辰喪葬
- 宗教等

日常生活

- 衣
- 食
- 住
- 行
- 其他日常生活

訟事

交際
- 姓氏

商業

- 經商行業
- 經營交易
- 賬目度量衡

交通

文化教育

- 學校
- 教室文具
- 讀書識字
- 寫字

文體活動

- 遊戲玩具
- 體育
- 武術舞蹈
- 戲劇

政治
位置

**詞性**

動作

- 一般動作
- 心理活動
- 語言動作

代詞等
形容詞
副詞介詞等
量詞
附加成分

數字
- 數目
- 約數
- 號碼
- 數量
- 數字俗語

歇後語

**情感**

- 褒義
- 貶義
- 戲謔

**其他**

**Reply by Guanchishan** [reply]
之後在這個文檔裏更新：https://github.com/ztl8702/zingzeu-data/commit/b7decfb8407c7149bdbcf3a1b7aa9e538947091a

**Comment by ztl8702** [comment]
感谢提供分类信息。 @Guanchishan 

已加到需求文档 https://shimo.im/docs/35c397ff76b647b1#anchor-8d77
